ocpmodels/common/ase_utils.py

```diff
@@ -45,11 +46,8 @@ def load_pretrained(self, checkpoint_path):
 
     def calculate(self, atoms, properties, system_changes):
         Calculator.calculate(self, atoms, properties, system_changes)
-        # TODO: do we want to hash the atoms object to store preprocessed data,
-        # or discard entirely for each step. Storage issues likely with little
-        # gain.
-        batch = self.trainer.dataset.ase_atoms_to_batch(atoms)
-        predictions = self.trainer.predict(batch, verbose=False)
+        batch = TrajectoryDataset.ase_atoms_to_batch(atoms)
```

Comment:
`TrajectoryDataset.ase_atoms_to_batch` still uses the old preprocessing code and is going to be removed soon.

- Add the following imports:
```python
from ocpmodels.preprocessing import AtomsToGraphs
from torch_geometric.data import Batch
```
- Initialize the preprocessor in `__init__`:
```python
self.a2g = AtomsToGraphs(
    max_neigh=12,
    radius=6,
    dummy_distance=7,
    dummy_index=-1,
    r_energy=True,
    r_forces=True,
    r_distances=False,
)
```
- And replace the above lines with:
```python
data_object = self.a2g.convert(atoms)
batch = Batch.from_data_list([data_object])
```

Reply:
Changed r_energy and r_forces to False since atoms objects being provided here will not have these attributes.